lib/turbolinks.rb

```diff
@@ -7,19 +7,27 @@
 require 'turbolinks/redirection'
 
 module Turbolinks
+  module ControllerConcern
```

Comment:
> This is public now. Should we document what behavior will change in people's controllers or link them to documentation where they can see that?

I don't think there's enough "substance" in Turbolinks to need proper rdocs (maybe because I'm lazy :P). However, I'm planning a complete revamp of the README in the next few weeks with more/better docs.

> I also don't like the Concern suffix here. Would `Turbolinks::Controller` be too broad?

@matthewd suggested that too. IMO it's confusing since that module isn't a controller. It's "something you add to your controller". So maybe `Turbolinks::ControllerBehavior`, `Turbolinks::ControllerFeatures`, ...?